src/config/types.js

```diff
@@ -135,7 +135,7 @@ export const MANAGEMENT = {
   CLUSTER_ROLE_TEMPLATE_BINDING: 'management.cattle.io.clusterroletemplatebinding',
   FEATURE:                       'management.cattle.io.feature',
   GROUP:                         'management.cattle.io.group',
-  KONTANIER_DRIVER:              'management.cattle.io.kontainerdriver',
+  KONTAINER_DRIVER:              'management.cattle.io.kontainerdriver',
```

Comment:
API?

Reply:
This should be made in [dashboard](https://github.com/rancher/dashboard/blob/2345527a916cde3909a5ca0ceae3b7b36f593603/config/types.js#L146) (among other things, we don't have any of that).